frontend/src/modules/organization/components/list/organization-list-table.vue

```diff
@@ -136,7 +136,7 @@
                           v-if="scope.row.headline || scope.row.description"
                           class="text-sm h-full flex items-center text-gray-900"
                         >
-                          {{ scope.row.headline || scope.row.description }}
+                          {{ truncateText((scope.row.headline || scope.row.description)) }}
```

Comment:
Could you also pass the `characters` and `suffix` so that maximum characters is (150) and suffix is `...` (so that the user can understand that there is more content there)?

Previously I wrote 250 in the issue, but 150 is more suited. Updated in the issue as well